### ISSUE TITLE
Allow adding of files to existing folders

### ISSUE THREAD

**Issue by dlangenk** [biigle/user-storage#13]
If you already created a directory it seems impossible to add a new file to it.

**Comment by mzur** [comment]
It should be possible to add files to the same directory in another storage request. Or did you mean something different?

**Comment by dlangenk** [comment]
No I think this is the same thing I meant. But if I go to https://biigle.de/storage-requests/create I do not see any directories I previously created and I can only add new directories.

**Comment by mzur** [comment]
> But if I go to https://biigle.de/storage-requests/create I do not see any directories I previously created and I can only add new directories.

That's right. But you can create a new directory with the same name than the one of a previous storage request. Once the request is approved, the files will be available in the same directory.

**Comment by dlangenk** [comment]
Is it possible to load the current file system structure? That would simplify things and not be that prone to typos. But good that it works.

**Comment by mzur** [comment]
IMO this would be confusing, as the displayed directories do not (have to) belong to the new storage request. What about suggestions for existing directory names that are displayed as a typeahead in the input field for the new directory name?

**Comment by dlangenk** [comment]
Would be okayish but if you don't know that you can create already existing folders there is no benefit (the real world experience tells you that this should not be possible). What about graying existing directories out or something.

**Comment by mzur** [comment]
That could work. We can leave this issue open and discuss the final implementation based on a first mockup.
